emission/analysis/classification/inference/mode.py

```diff
@@ -79,29 +146,39 @@ def getModeQuery(groundTruthMode):
 
   @staticmethod
   def getSectionQueryWithGroundTruth(groundTruthMode):
-    return {"$and": [{'type': 'move'},
+    # return {"$and": [{'type': 'move'},
+    return {"$and": [{"$or": [{'type': { '$exists' : False }}, {'type' : 'move'}]},        
```

Comment:
Yes, I had originally kept this in because I was running it both with the old datafile and the new one. Later on I stop doing this, so I take it out of other queries, but I left some of them in with the intention to clean them up later.